bitnami/mongodb-sharded/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: common
   repository: https://charts.bitnami.com/bitnami
-  version: 1.7.0
-digest: sha256:6786bc7fdfc6e4038fd28819c2b07339f232282dbaeab21de8064aaa6814a8d1
-generated: "2021-07-07T15:32:21.159182482Z"
+  version: 1.7.1
+digest: sha256:d05549cd2eb5b99a49655221b8efd09927cc48daca3fa9f19af0257a11e5260f
```

Comment:
This changed when installing the required other packages, not sure if I should keep it or not 🤷

Reply:
We can remove this file from the PR, no problem